### DISCUSSION TITLE
About the student's hit rate

### DISCUSSION THREAD

**Discussion by neko25nky** [lonqie/SchaleDB#88] [Questions] [answered]
Nice to meet you, I am translating using DeepL, so I apologize for any unnatural English.

I have a question about the hit rate of students.
I think the hit rate can be calculated by 700/(evasion value - hit value +700), but if I try to calculate it with Shiroko (swimsuit) and Tsubaki, I get
700/(1416 - 660 + 700) = 0.4807692308 (48.07...%)
but the site shows 46.86%.
Where does this difference come from?
![スクリーンショット 2024-08-28 191000](https://github.com/user-attachments/assets/f3427518-9704-42ca-8431-054131ea8d34)

**Comment by lonqie** [comment, marked as answer]
That formula is wrong, the constant should be 2000/3 or 666.666... not 700.
Source: https://docs.google.com/document/d/1Ui0FRgG8h8RB1-cA639F3ETDrVM7q9FKftg4vBKBAAQ

**Reply by neko25nky** [reply]
Thank you for your response.
At that URL, the Japanese wiki (https://bluearchive.wikiru.jp/?cmd=read&page=%E3%82%B3%E3%83%A1%E3%83%B3%E3%83%88%2F%E6%83%85%E5%A0%B1%E6%8F%90%E4%BE%9B%E6%8E%B2%E7%A4%BA%E6%9D%BF) as a reference for calculating the hit rate, but within that site it is said that “hit rate = 700/(evasion value - hit value + 700)”. With an image (https://bluearchive.wikiru.jp/?plugin=attach&refer=%E6%83%85%E5%A0%B1%E6%8F%90%E4%BE%9B%E6%8E%B2%E7%A4%BA%E6%9D%BF&openfile=%E3%83%96%E3%83%AB%E3%82%A2%E3%82%AB%E5%91%BD%E4%B8%AD%E7%8E%87%E6%A4%9C%E8%A8%BC.png).

**Reply by lonqie** [reply]
Yes, it seems the source in that document is outdated. It was previously thought to be 700, but further testing was done and found the constant to actually be 2000/3. This testing happened in a discord server so I am unable to link it, however the formula was verified using code decompilation so we can be reasonably certain it is accurate.

**Reply by neko25nky** [reply]
So it was. The Japanese wiki has always assumed “700/(hit value - avoidance value)700”, but I didn't know there was something verified by decompiling the code.
Sorry for the trouble.
